web/src/Routes.tsx

```diff
@@ -0,0 +1,19 @@
+import React, {lazy, Suspense} from 'react';
+import {Switch, Route} from 'react-router-dom';
+import Loading from 'components/common/Loading';
+
+// Customer Pages
+const CustomerExplorePage = lazy(() => import('components/customer/explore'));
+// Merchant Pages
+const MerchantLandingPage = lazy(() => import('components/merchant/landing'));
+
+const Routes: React.FC = () => (
+  <Suspense fallback={<Loading />}>
+    <Switch>
+      <Route exact path="/" component={CustomerExplorePage} />
```

Comment:
why not path="/customer"?

Reply:
@ifhuang Hmm then wdyt "/" should be? I was thinking like how most listing sites have customer facing pages at "/" and business facing pages under "/businesses" or something, that's why I decided to go with this split.

Reply:
ic, make sense